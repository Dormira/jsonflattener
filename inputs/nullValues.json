{
	"a": 1,
	"null": ,
	"b": true,
	"c": {
		"c" : {},
		"e": {}
	}
}
