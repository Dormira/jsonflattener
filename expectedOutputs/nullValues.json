Couldn't load json file inputs/nullValues.json: Expecting value: line 3 column 10 (char 20)